streams/src/main/java/org/apache/kafka/streams/state/internals/ChangeLoggingWindowBytesStore.java

```diff
@@ -64,10 +64,8 @@ public void put(final Bytes key, final byte[] value) {
 
     @Override
     public void put(final Bytes key, final byte[] value, final long timestamp) {
-        if (key != null) {
-            bytesStore.put(key, value, timestamp);
-            changeLogger.logChange(WindowStoreUtils.toBinaryKey(key, timestamp, maybeUpdateSeqnumForDups(), innerStateSerde), value);
-        }
+        bytesStore.put(key, value, timestamp);
```

Comment:
Note this removal is intentional since originally we add the check un-carefully to go around the real issue of Bytes.wrap; now we need to remove it to really trigger the NPE in our unit tests.